config/rbac/role.yaml

```diff
@@ -37,12 +37,6 @@ rules:
     - persistentvolumeclaims/finalizers
   verbs:
     - '*'
```

Comment:
Could it break statefulset rollout update process? 

 It uses delete operation for pods.

https://github.com/VictoriaMetrics/operator/blob/master/controllers/factory/k8stools/sts.go#L201

Reply:
Yes, you're right!
Sorry I forgot about this, fixed other insufficient roles in the latest commit, please check.